server/lib/update-configuration.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Wazuh app - Module to update the configuration file
+ * Copyright (C) 2018 Wazuh, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * Find more information about this on the LICENSE file.
+ */
+import fs from 'fs';
+import readline from 'readline';
+import path from 'path';
+
+export function updateConfigurationFile(req) {
```

Comment:
This function must be async, we don't want read lines using events. Even if we do that, we want promises always, so you may want to wrap a callback function into promise function.